src/store.ts

```diff
@@ -89,6 +109,8 @@ export const initialStoreState: C4CState = {
   mapGeoDataState: initialMapGeoDataState,
   teamState: initialTeamState,
   availableTeamsState: initialAvailableTeamsState,
+  siteState: initialSiteState,
+  adoptedSitesState: initialProtectedSiteState,
```

Comment:
❓ Would it make sense to have `adoptedSites` be a field of `SiteReducerState` instead of having a separate sub-state?

Reply:
The adopted sites is a protected route while the other stewardship activities and site data are not. Meaning we want to be able to dispatch the public site actions without dispatching the adopted sites if the user is not logged in. Also because they're in different api clients have to have separate thunk actions (at least as far as I know).